### ISSUE TITLE
Accessibility client can sit on walls

### ISSUE THREAD

**Issue by Restioson** [NucleoidMC/Electric-Floor#1]
Barrier blocks should be added around the edges so that accessibility client users cannot jump onto them and circumvent the entire game.
